src/v0/destinations/fb_custom_audience/util.js

```diff
@@ -129,6 +129,7 @@ const ensureApplicableFormat = (userProperty, userInformation) => {
 
 const getUpdatedDataElement = (dataElement, isHashRequired, eachProperty, updatedProperty) => {
   let tmpUpdatedProperty = updatedProperty;
+  // if hashRequired is true and the property name is not (MADID or EXTERN_ID) hash the value if it is defined else send null
```

Comment:
this comment is not helpful as it is direct translation of code, you can remove it, it is fine.